sdk/openai/openai/review/openai.api.md

```diff
@@ -129,10 +163,34 @@ export interface GetEmbeddingsOptions extends RequestOptions {
 }
 
 // @public
+export type ImageGenerationResponseFormat = string;
```

Comment:
Why is this alias being created? Are we expecting this type to change in the future?

Reply:
Great question! This came from the TS emitter, seems like the type is supposed to actually be one of these enum values:

```
enum ImageGenerationResponseFormat {
  @doc("Image generation response items should provide a URL from which the image may be retrieved.")
  Url: "url",

  @doc("Image generation response items should provide image data as a base64-encoded string.")
  Base64: "b64_json",
}
```

@qiaozha @MaryGao - do we not support enums yet?

Reply:
@glharper yeah looks like currently we normalize to 'string' and then comment the options, from the generated types:

```ts
/** The state of a job or item. */
/** "notRunning", "running", "succeeded", "canceled", "failed" */
export type AzureOpenAIOperationState = string;
/** The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024. */
/** "256x256", "512x512", "1024x1024" */
export type ImageSize = string;
/** The format in which the generated images are returned. */
/** "url", "b64_json" */
export type ImageGenerationResponseFormat = string;
```

Reply:
@glharper looks like unless the enum is decorated as [@fixed](https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/decorators#@Azure.Core.fixed) we generate it as a string since it could expand later. Not sure if we have a better story for this at the moment.

Reply:
@xirzec Understood, thanks for the explanation.